demo/src/app/components/+datepicker/datepicker-section.list.ts

```diff
@@ -131,6 +132,14 @@ export const demoComponentContent: ContentSection[] = [
         component: require('!!raw-loader?lang=typescript!./demos/reactive-forms/reactive-forms.component.ts'),
         html: require('!!raw-loader?lang=markup!./demos/reactive-forms/reactive-forms.component.html'),
         outlet: DemoDatepickerReactiveFormsComponent
+      },
+      {
+        title: 'Visibility methods',
+        anchor: 'visibility-methods',
+        component: require('!!raw-loader?lang=typescript!./demos/visibility-methods/visibility-methods.ts'),
+        html: require('!!raw-loader?lang=markup!./demos/visibility-methods/visibility-methods.html'),
+        description: `<p>Was used <code>outsideClick</code> property to show how to work methods</p>`,
```

Comment:
change description to : You can manage datepicker's state by using its `show()`, `hide()` and `toggle()` methods